earthly/bash/duplicated-scripts.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Takes a single argument, the directory to check.
+
+# Get our includes relative to this file's location.
+source "/scripts/include/colors.sh"
+
+rc=0
+
+# Create an associative array to store the file contents and their corresponding filenames
+declare -A files
+
+# Loop through all files with the .sh extension recursively
```

Comment:
Is this "# Loop through all files......" intentionally duplicated from the line below?

Reply:
Fixed, thanks.